README.md

```diff
@@ -13,13 +13,13 @@ The [Education Initiative](https://chanzuckerberg.com/education/)'s design syste
 
 ### Helpful commands
 
-Description | Command
------------ | -------
-Install dependencies | `npm install && npx lerna bootstrap`
-Run linter | `npm run lint`
-Run linter and fix all fixable issues | `npm run lint:fix`
-Build packages | `npm run build`
-Run the component generator | `npx plop`
+| Description                           | Command                              |
+| ------------------------------------- | ------------------------------------ |
+| Install dependencies                  | `npm install && npx lerna bootstrap` |
+| Run linter                            | `npm run lint`                       |
+| Run linter and fix all fixable issues | `npm run lint:fix`                   |
+| Build packages                        | `npm run build`                      |
+| Run the component generator           | `npx plop`                           |
```

Comment:
hmm my prettier did this, but doesn't look like it changed anything in the rich diff

Reply:
Just easier to read the raw file I guess.

Reply:
WAT I've never seen prettier change markdown tables before 🤔 

Kind of cool, though